build.sbt

```diff
@@ -163,6 +163,8 @@ lazy val interopTests = Project(id = "interop-tests", base = file("interop-tests
     ReflectiveCodeGen.generatedLanguages := Seq("Scala", "Java"),
     ReflectiveCodeGen.extraGenerators := Seq("ScalaMarshallersCodeGenerator"),
     ReflectiveCodeGen.codeGeneratorSettings ++= Seq("server_power_apis"),
+    // grpc 1.53.0 brings in extra unnecessary proto files that cause build issues
+    PB.generate / excludeFilter := new SimpleFileFilter(f => f.getAbsolutePath().contains("envoy/")),
```

Comment:
Drop the `/`

Reply:
made the change - I suspect there might be other issues with building this module on Windows but I think this PR should not now make it worse